css/global/variables.css

```
@@ -0,0 +1,37 @@
+:root {
+  --c-basic-extra-dark: #2d3440;
+  --c-basic-dark: #565c66;
+  --c-basic-neutral: #b9b9b9;
+  --c-basic-light: #e7e7e7;
+  --c-basic-extra-light: #fcfcfc;
+
+  --c-special-extra-bright: #ff2f64;
+  --c-special-bright: #ff4a78;
+  --c-special-neutral: #ff7799;
+  --c-special-light: #feafc3;
+  --c-special-extra-light: #ffcbd8;
+  --c-special-blue: #69a9ff;
+  --c-special-yellow: #fcc850;
+
+  --c-status-error: #ea5454;
+  --c-status-warning: #fd912e;
+  --c-status-success: #00e484;
+}
+
+:root {
+  --c-basic-extra-light-30: rgba(252, 252, 252, 0.3);
+  --c-basic-extra-light-50: rgba(252, 252, 252, 0.5);
+}
+
+:root {
+  --shadow-sm: 0px 4px 12px rgba(45, 52, 64, 0.1);
```

Если ноль, то без единиц измерения
```suggestion
  --shadow-sm: 0 4px 12px rgba(45, 52, 64, 0.1);
```